app/assets/images/index.js

```diff
@@ -0,0 +1,43 @@
+import BackArrow from './backArrow.png';
+import BackgroundAtRisk from './backgroundAtRisk.png';
+import BoxCheckedIcon from './boxCheckedIcon.png';
+import boxUncheckedIcon from './boxUncheckedIcon.png';
+import Close from './closeIcon.png';
+import Export from './export.png';
+import ForeArrow from './foreArrow.png';
+import HomeScreenBackgroundOverlay from './homeScreenBackgroundOverlay.png';
+import LaunchScreen1 from './launchScreen1.png';
+import LaunchScreen2 from './launchScreen2.png';
+import LaunchScreen3 from './launchScreen3.png';
+import LaunchScreenBackground from './launchScreenBackground.png';
+import LaunchScreenBackgroundOverlay from './launchScreenBackgroundOverlay.png';
+import Newspaper from './newspaper.png';
+import PrivateKitQRCode from './privatekit_qrcode.png';
+import SaveIcon from './saveIcon.png';
+import UserGreen from './user-green.png';
+import Welcome1 from './welcome1.png';
+import WWW from './www.png';
+
+const Images = {
+  BackArrow,
+  BackgroundAtRisk,
+  BoxCheckedIcon,
+  boxUncheckedIcon,
+  Close,
+  Export,
+  ForeArrow,
+  HomeScreenBackgroundOverlay,
+  LaunchScreen1,
+  LaunchScreen2,
+  LaunchScreen3,
+  LaunchScreenBackground,
+  LaunchScreenBackgroundOverlay,
+  Newspaper,
+  PrivateKitQRCode,
+  SaveIcon,
+  UserGreen,
+  Welcome1,
+  WWW,
+};
+
+export default Images;
```

Comment:
use named exports everywhere.

Reply:
Out of curiosity, why is this? Is it to maintain code consistency or is there a technical reason?

Reply:
I think we've been leaning towards named exports for

1. Consistency
2. IDE integration (better autocomplete, etc)

Reply:
Even for react components?